tests/trainer/connectors/test_callback_connector.py

```diff
@@ -211,6 +211,21 @@ def test_attach_model_callbacks_override_info(caplog):
     trainer.model = model
     cb_connector = CallbackConnector(trainer)
     with caplog.at_level(logging.INFO):
-        cb_connector._attach_model_callbacks()
+        cb_connector._attach_model_callbacks(model)
 
     assert "existing callbacks passed to Trainer: EarlyStopping, LearningRateMonitor" in caplog.text
+
+
+def test_attach_model_callbacks_support_logging(tmpdir):
```

Comment:
I believe the test_hooks.py is sufficient and no extra test needs to be added.

Reply:
yeah but no tests were failing, right?

Reply:
oh ! okay. got it.